src/web_app/src/config/utils.ts

```diff
@@ -0,0 +1,57 @@
+import {Link} from "../types/d3Types";
+import * as Types from "../types/data";
+import * as d3Types from "../types/d3Types";
+
+export function getRandomColor() {
+    return "#" + Math.floor(Math.random() * 16777215).toString(16)
```

Comment:
Some colors have a terrible contrast with the white background, if I well understand the code this could be fixed by changing this function, would it be a hard fix  ?